scu_builders.py

```diff
@@ -337,13 +424,21 @@ def generate_scu_files(max_includes_per_scu):
     process_folder(["modules/gdscript"])
     process_folder(["modules/gdscript/editor"])
     process_folder(["modules/gdscript/language_server"])
+    process_folder(["modules/webxr"])
+    process_folder(["modules/websocket"])
+
+    # The second list here is a list of subfolders that are not required to be built.
+    process_folder_recursive("thirdparty/jolt_physics/Jolt", ["NodeCodec", "TriangleCodec", "ConstraintPart"])
```

Comment:
How come these (seemingly header-only) folders need to be excluded?

Reply:
If they are header only, they don't need to be excluded, you are correct, I didn't spot they were header only. Will change. :+1: